source/scss/themes/default/_theme.variables.scss

```diff
@@ -1,33 +1,31 @@
-@import "tools.mixins";
-
 /*------------------------------------*\
     $VARIABLES
 \*------------------------------------*/
 
 /**
  * Grid & Baseline Setup
  */
-$ff-fontpx: 16; // Font size (px) baseline applied to <body> and converted to %.
```

Comment:
are you waiting to hear chris and laura w/r/t baseline font size?

Reply:
Yeah, at some point, but it's not a blocker.